src/ctia/properties.clj

```diff
@@ -20,13 +22,40 @@
 (defonce properties
   (atom {}))
 
+(defn default-store-properties [store]
+  {(str "ctia.store." store) s/Str})
+
+(defn sql-store-impl-properties [store]
+  {(str "ctia.store.sql." store ".classname") s/Str
+   (str "ctia.store.sql." store ".subprotocol") s/Str
+   (str "ctia.store.sql." store ".subname") s/Str
+   (str "ctia.store.sql." store ".delimiters") s/Str})
+
+(defn es-store-impl-properties [store]
+  {(str "ctia.store.es." store ".uri") s/Str
```

Comment:
Here we have `uri` and also `port`, `cluster name`, `indexname` and `refresh` while apparently `port` is never used and is part of the `uri` (`http://localhost:9200`).

For me an uri is something that could contain more informations than just host and port, for example: `http://user:password@elasticsearch:9200/index_name` for example.

Couldn't we rename `uri` as `host` and assume the protocol to be `http` and use `port`?

I know this is certainly an older issue not directly related to this PR. So this shouldn't be a blocker.